circle.yml

```diff
@@ -1,7 +1,10 @@
 machine:
   node:
-    version: 6.11.1
```

Comment:
This get's used by our lambdas. They don't have support >=6.x.x so should we be setting our CI version to this?

Reply:
Alright, cool, I'll change it back. Good spot